syntax = "proto3";

message FanTimingConfig{

    string  active = 1;   

    string  baseTime = 2; 

    string  endTime = 3;  

    int32 interval = 4;    

    int32 openMin = 5;   

}